src/libraries/System.Configuration.ConfigurationManager/tests/System/Configuration/BinaryFormatterDeprecationTests.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Do we also care about adding a test case using reflection as well? 

cc: @GrabYourPitchforks

Reply:
I think in general we don't care about binary formatter usage if it is called explicitly via reflection by a user.